api/src/main/java/datawave/webservice/results/datadictionary/DefaultDataDictionary.java

```diff
@@ -26,6 +28,7 @@
 import io.protostuff.Message;
 import io.protostuff.Output;
 import io.protostuff.Schema;
+import org.apache.avro.data.Json;
```

Comment:
Is this an unused import?  I didn't load it up in my IDE but I'm not seeing this used below.